SPEC.md

```diff
@@ -85,33 +85,54 @@ This specification outlines the requirements for botbuilder samples.
 ```markdown
 <INSERT AT MOST ONE PARAGRAPH DESCRIPTION OF WHAT THIS SAMPLE DOES> 
 
+# Table of Contents
+<LINKS TO SECTIONS OF THE README>
+
 # Concepts introduced in this sample
 <DESCRIPTION OF THE CONCEPTS>
 
+#Services used in this sample
+<INTRODUCE SERVICES>
 
 # To try this sample
--	<STEPS TO CLONE REPO AND GET SETUP>
+
 ## Prerequisites
--	<REQUIRED TOOLS, VERSIONS>
--	<STEPS TO GET SET UP WITH THE SAMPLE. E.g. RUN AN INCLUDED SCRIPT OR MANUALLY DO SOMETHING ETC>
+
+###	Required Tools
+-   <Required TOOLS WITH MINIMUM VERSION NUMBERS>
+
+### Clone the repo
+-	<STEPS TO CLONE REPO AND GET SETUP>
 
 NOTE: <ANY NOTES ABOUT THE PREREQUISITES OR ALTERNATE THINGS TO CONSIDER TO GET SET UP>
 
-## Visual studio
+## Ways to get started with Bot Framework Samples
+
+### MSBot
```

Comment:
>MSBot [](start = 4, length = 5)

I believe in Vishwac's mind, msbot has two methods : Clone and deploy.  This is probably meant to be "msbot clone".